packages/flutter_tools/templates/template_manifest.json

```diff
@@ -54,7 +54,6 @@
         "templates/app_shared/ios.tmpl/Runner.xcodeproj/project.xcworkspace/contents.xcworkspacedata",
         "templates/app_shared/ios.tmpl/Runner.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist",
         "templates/app_shared/ios.tmpl/Runner.xcodeproj/project.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings",
-        "templates/app_shared/ios.tmpl/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme",
```

Comment:
Drive-by cleanup since it confused my while renaming the existing file; this file doesn't actually exist any more (I assume since the ObjC/Swift split).

Reply:
I think it does?

https://github.com/flutter/flutter/blob/master/packages/flutter_tools/templates/app_shared/ios.tmpl/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme

Reply:
Hm, maybe I confused myself when looking locally. Since we also have -swift and -ios versions, I suspect this is a mistake (and is being overwritten maybe?) I'll investigate more.

Reply:
It did exist, but shouldn't since it will overwrite the specific versions. I've removed the file as well now.